opentreemap/importer/tasks.py

```diff
@@ -118,6 +119,15 @@ def run_import_event_validation(import_type, import_event_id, file_obj):
         return
 
 
+@transaction.atomic
+def _update_ie_status(ie):
```

Comment:
The name `_update_ie_status` may be too general for such a specific action. Since we are conditionally updating the status to a specific value, `_update_status_to_verifying_if_not_past` is a little wordy, but more self-describing.